app/src/main/java/com/example/poste/models/User.java

```diff
@@ -177,17 +176,17 @@ public Post getPost(String postId) {
     }
 
     /**
-     * Returns a copy of the list of the user's folders. This is a copy, so modifying the returned
-     * list will not modify the user's folders.
-     * @return a copy of the list of the user's folders.
+     * Returns List of the user's folders.
+     * @return a List of the user's folders.
      */
     public List<Folder> getFolders() {
-        return new ArrayList<>(folders);
+        return folders;
```

Comment:
I like the idea of immutable return values here (returning a copy, instead of the actual list) -- however, I think this was causing some confusion and we were attempting to modify the returned lists of folders / posts. Until we can get on the same page and refactor, this will solve some issues.